tidb-cloud/serverless-driver.md

```diff
@@ -1,12 +1,18 @@
 ---
 title: TiDB Cloud Serverless Driver (Beta)
-summary: Learn how to connect to TiDB Serverless from serverless and edge environments over HTTP.
+summary: Learn how to connect to TiDB Serverless from serverless and edge environments.
 aliases: ['/tidbcloud/serverless-driver-config']
 ---
 
-# TiDB Cloud Serverless Driver (Beta)
+## Why use TiDB Cloud Serverless Driver (Beta)
```

Comment:
We should keep the original title and opening paragraph. Correct me if the content is not appropriate from your point of view.

```suggestion
# TiDB Cloud Serverless Driver (Beta)

This document describes what is TiDB Cloud serverless driver and how to use it to connect to TiDB Serverless from serverless and edge environments.

## Why use TiDB Cloud serverless driver
```

Reply:
I think it looks redundant for me and contains no useful info.

Reply:
You can remove the paragraph, but the title stays. Each document must have one single L1 heading for the purpose of SEO.

Reply:
It's okay for me then.